Signal/src/Models/TSUnreadIndicatorInteraction.m

```diff
@@ -50,6 +50,11 @@ - (BOOL)isDynamicInteraction
     return YES;
 }
 
+- (OWSInteractionType)interactionType
+{
+    return OWSInteractionType_UnreadIndicator;
```

Comment:
All `TSInteraction` now has a `interactionType` method so that we can different distinguish interaction and cell types without using `isKindOfClass:`.